Gemfile.lock

```diff
@@ -51,42 +65,6 @@ PATH
       proxifier (~> 1.0)
       syslog-logger (~> 1.6)
       uuidtools (~> 2.1.5)
-    chef (15.0.211-universal-mingw32)
```

Comment:
you've lost the windows Gemfile.lock stuff here.  this is where `rake dependences:update_gemfile_lock` can get used to re-create them.

Reply:
Done, and also updated to the released chef-core gem.